controllers/discordactions.js

```diff
@@ -121,7 +120,41 @@ const addGroupRoleToMember = async (req, res) => {
     return res.boom.badImplementation(INTERNAL_SERVER_ERROR);
   }
 };
+/**
+ * Patch Update user nickname
+ * @param req {Object} - Express request object
+ * @param res {Object} - Express response object
+ */
+const changeNicknameOfUsers = async (req, res) => {
+  try {
+    const userData = req.userData;
 
+    const { discordId, username } = userData;
```

Comment:
Not all users in the DB have `discordId`. how do we handle that 🤔

Reply:
we have checkInVerfified middleware for the same. if you are verified then this API will be called then it will add discordId in the userData.